_data/retail.yml

```
@@ -64,36 +64,51 @@ websites:
       - sms
       - totp
     doc: https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c
+    regions:
+      - ca
+      - us
+      - mx
 
   - name: Big W
     url: https://www.bigw.com.au/
     img: bigw.png
     facebook: BIGWaustralia
+    regions:
+      - au
 
   - name: Bonobos
     url: https://bonobos.com
     img: bonobos.png
     twitter: Bonobos
     facebook: bonobos
+    regions:
+      - us
```

```suggestion
      - us
      - ca
      - mx
      - au
      - nz
      - jp
      - pe
      - mg
```
Bonobos ships to [Canada, Mexico, Australia, New Zealand, Japan, Peru, and Madagascar](https://help.bonobos.com/s/article/international---where-do-you-ship-internationally).